efel/cppcore/LibV1.cpp

```diff
@@ -271,7 +272,7 @@ int LibV1::firing_rate(mapStr2intVec& IntFeatureData,
     return nSize;
   else {
     vector<double> stimStart, stimEnd, peakVTime, firing_rate;
-    double lastAPTime;
+    double lastAPTime = 0.;
```

Comment:
That is correct, unless stimStart[0] == lastAPTime == 0.0, at which point we're in NaN land.

I can either add an assert, or a return -1, what do you prefer?